distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -30,80 +29,62 @@
 import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.useOwnExamBundlesStartLevel;
-import static com.jayway.restassured.RestAssured.get;
 
 import java.io.File;
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Collection;
-import java.util.Dictionary;
-import java.util.Enumeration;
 import java.util.HashMap;
-import java.util.Hashtable;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
 import javax.inject.Inject;
 
-import org.apache.commons.lang.StringUtils;
 import org.apache.karaf.features.FeaturesService;
-import org.apache.karaf.shell.osgi.BlueprintListener;
-import org.apache.karaf.shell.osgi.BlueprintListener.BlueprintState;
 import org.junit.Rule;
 import org.ops4j.pax.exam.Option;
 import org.ops4j.pax.exam.karaf.options.LogLevelOption;
-import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
-import org.osgi.framework.Constants;
 import org.osgi.framework.InvalidSyntaxException;
-import org.osgi.framework.ServiceReference;
-import org.osgi.service.cm.Configuration;
 import org.osgi.service.cm.ConfigurationAdmin;
-import org.osgi.service.cm.ConfigurationEvent;
-import org.osgi.service.cm.ConfigurationListener;
-import org.osgi.service.metatype.AttributeDefinition;
-import org.osgi.service.metatype.MetaTypeInformation;
 import org.osgi.service.metatype.MetaTypeService;
-import org.osgi.service.metatype.ObjectClassDefinition;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.osgi.util.OsgiStringUtils;
-
-import com.jayway.restassured.response.Response;
 
 import ddf.catalog.CatalogFramework;
 import ddf.catalog.source.CatalogProvider;
 import ddf.catalog.source.FederatedSource;
+import ddf.common.test.AdminConfig;
 import ddf.common.test.PaxExamRule;
+import ddf.common.test.PostTestConstruct;
+import ddf.common.test.ServiceManager;
 
 /**
  * Abstract integration test with helper methods and configuration at the container level.
  */
 public abstract class AbstractIntegrationTest {
 
-    protected static final int CONFIG_UPDATE_WAIT_INTERVAL = 5;
-
     protected static final Logger LOGGER = LoggerFactory.getLogger(AbstractIntegrationTest.class);
 
-    protected static final String LOG_CONFIG_PID = "org.ops4j.pax.logging";
+    protected static final String LOG_CONFIG_PID = AdminConfig.LOG_CONFIG_PID;
+
+    protected static final String LOGGER_PREFIX = AdminConfig.LOGGER_PREFIX;
 
-    protected static final String LOGGER_PREFIX = "log4j.logger.";
+    protected static final String DEFAULT_LOG_LEVEL = AdminConfig.DEFAULT_LOG_LEVEL;
 
     protected static final String KARAF_VERSION = "2.4.3";
     
-    protected static final int ONE_MINUTE_MILLIS = 60000;
+    protected static final long ONE_MINUTE_MILLIS = TimeUnit.MINUTES.toMillis(1);
```

Comment:
The others that you'd created and I'd moved aren't specifically used through `AbstractIntegrationTest` anymore either. What are your thoughts about removing those?